examples/models/llama2/README.md

```diff
@@ -91,6 +91,20 @@ If you want to deploy and run a smaller model for educational purposes. From `ex
     python -m examples.models.llama2.tokenizer.tokenizer -t tokenizer.model -o tokenizer.bin
     ```
 
+
+## (Optional) Finetuning
+
+If you want to finetune your model based on a specific dataset, PyTorch provides [TorchTune](https://github.com/pytorch/torchtune) - a native-Pytorch library for easily authoring, fine-tuning and experimenting with LLMs.
+
+Once you have [TorchTune installed](https://github.com/pytorch/torchtune?tab=readme-ov-file#get-started) you can finetune Llama2 7B model using LoRA on a single GPU, using the following command:
+
+
+tune run lora_finetune_single_device \
+--config llama2/7B_lora_single_device \
+checkpointer.checkpoint_dir=<path_to_checkpoint_folder>  \
+tokenizer.path=<path_to_checkpoint_folder>/tokenizer.model
```

Comment:
nit: Should wrap this in a code block (sorry suggestions doesnt allow me to do that)